webpack.config.js

```diff
@@ -27,6 +27,7 @@ module.exports = (env, argv) => {
     plugins: [
       new webpack.NormalModuleReplacementPlugin(new RegExp(path.resolve(__dirname, 'lib/storage.js')), path.resolve(__dirname, 'browser-version/lib/storage.js')),
       new webpack.NormalModuleReplacementPlugin(new RegExp(path.resolve(__dirname, 'lib/customUtils.js')), path.resolve(__dirname, 'browser-version/lib/customUtils.js')),
+      new webpack.NormalModuleReplacementPlugin(new RegExp('byline'), path.resolve(__dirname, 'browser-version/lib/byline.js')),
```

Comment:
No need to use the `RegExp` ctor, we should use directly `/byline/`  (standardjs)